### ISSUE TITLE
Does not support angular schematics

### ISSUE THREAD

**Issue by pickfire** [ionic-team/angular-toolkit#145]
From an existing angular project. After running `ng add @ionic/angular`.
```
Installing packages for tooling via yarn.
yarn add v1.16.0
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.2.9: The platform "linux" is incompatible with this module.
info "fsevents@1.2.9" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
[4/4] Building fresh packages...

success Saved lockfile.
success Saved 3 new dependencies.
info Direct dependencies
└─ @ionic/angular@0.2.2
info All dependencies
├─ @ionic/angular@0.2.2
├─ @ionic/core@0.2.2
└─ ionicons@4.0.0-18
Done in 16.14s.
Installed packages for tooling via yarn.
The package that you are trying to add does not support schematics. You can try using a different version of the package or contact the package author to add ng-add support.
```

```
> ng --version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 8.0.3
Node: 11.15.0
OS: linux x64
Angular: 8.0.1
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.800.3
@angular-devkit/build-angular     0.800.3
@angular-devkit/build-optimizer   0.800.3
@angular-devkit/build-webpack     0.800.3
@angular-devkit/core              8.0.3
@angular-devkit/schematics        8.0.3
@angular/cli                      8.0.3
@ngtools/webpack                  8.0.3
@schematics/angular               8.0.3
@schematics/update                0.800.3
rxjs                              6.4.0
typescript                        3.4.5
webpack                           4.30.0

```

**Comment by mhartington** [comment]
The ng-add schematic is part of `ionic/angular`, not this package. What appears to be happening is that you are adding ionic/angular to an angular 8 project, which at the time of this issue we did not support. We just released ionic/angular 4.7 which does add support for Angular 8.

**Comment by pickfire** [comment]
@mhartington But why does the update does not support `ng update @ionic/angular` but instead it needs to `yarn update ...` so many packages one by one?

By the way, the issue is not specific to Angular 8, it could be Angular 7.
